infra/docker-compose/docker-compose.yml

```diff
@@ -22,21 +22,23 @@ services:
       - --spring.config.location=classpath:/application.yml,file:/etc/feast/application.yml
 
   jupyter:
-    image: jupyter/minimal-notebook:619e9cc2fc07
+    image: gcr.io/kf-feast/feast-jupyter:latest
     volumes:
       - ./gcp-service-accounts/${FEAST_JUPYTER_GCP_SERVICE_ACCOUNT_KEY}:/etc/gcloud/service-accounts/key.json
-      - ./jupyter/startup.sh:/etc/startup.sh
     depends_on:
       - core
     environment:
       FEAST_CORE_URL: core:6565
       FEAST_ONLINE_SERVING_URL: online-serving:6566
       FEAST_BATCH_SERVING_URL: batch-serving:6567
       GOOGLE_APPLICATION_CREDENTIALS: /etc/gcloud/service-accounts/key.json
-      FEAST_REPOSITORY_VERSION: ${FEAST_REPOSITORY_VERSION}
     ports:
       - 8888:8888
-    command: ["/etc/startup.sh"]
+    command: >
+      bash -c "
+      git clone https://github.com/feast-dev/feast.git || true
```

Comment:
Is there a reason why can't we bake in the Feast repo into the docker image?

Reply:
The `latest` image will only be built on a release, which means that if we want to roll out the latest tutorials then we have to cut a release. Often I find myself updating docs/tutorials between releases, which is why I have pinned this `clone` to the branch instead. In theory we could build the `latest` image on every commit, but that image will go stale very quickly on the user's side (they have to repull the whole time).